src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/gtfs/StopTestsIT.java

```diff
@@ -0,0 +1,124 @@
+      .startMock();
+
+    waitApiListCount("/organizations", 1);
+    
+    createGtfsSettings(getOrganizationId(0));
+
+    waitApiListCount(String.format("/organizations/%s/transportStops", getOrganizationId(0)), 5); 
+  }
+
+  @After
+  public void afterClass() {
+    String organizationId = getOrganizationId(0);
+    getPtvMocker().endMock();
+    deletePtvSettings();
+    deleteGtfsSettings(organizationId);
+  }
+
+  @Test
+  public void testListStops() {
+    given()
+      .config(new RestAssuredConfig().jsonConfig(new JsonConfig(JsonPathConfig.NumberReturnType.DOUBLE)))
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/transportStops", getOrganizationId(0))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(5))
+      .body("id[1]", notNullValue())
+      .body("name[1]", is("Vanhala koulu"))
+      .body("lat[1]", is(61.7696611577961))
+      .body("lng[1]", is(27.1201016871884));
+  }
+  
+  @Test
+  public void testFindStop() {
+    String organizationId = getOrganizationId(0);
+    given()
+      .config(new RestAssuredConfig().jsonConfig(new JsonConfig(JsonPathConfig.NumberReturnType.DOUBLE)))
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/transportStops/{stopId}", organizationId, getOrganizationStopId(organizationId, 2))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id", notNullValue())
+      .body("name", is("Tenholahdentie  P"))
+      .body("lat", is(61.7005145626563))
+      .body("lng", is(27.2056931881733));
+
+  }
+  
+  @Test
+  public void testOrganizationStopsNotFound() throws InterruptedException {
+    String organizationId = getOrganizationId(0);
+    String incorrectOrganizationId = getOrganizationId(1);
+    String organizationStopId = getOrganizationStopId(organizationId, 0);
+    
+    String[] malformedIds = new String[] {"evil", "*", "/", "1", "-1", "~"};
+    assertFound(String.format("/organizations/%s/transportStops/%s", organizationId, organizationStopId));
+    assertEquals(5, countApiList(String.format("/organizations/%s/transportStops", organizationId)));
+    
+    for (String malformedId : malformedIds) {
+      assertNotFound(String.format("/organizations/%s/transportStops/%s", organizationId, malformedId));
+    }
+    
+    assertNotFound(String.format("/organizations/%s/transportStops/%s", incorrectOrganizationId, organizationStopId));
+    assertEquals(0, countApiList(String.format("/organizations/%s/transportStops", incorrectOrganizationId)));
+  }
+
+  private void createPtvSettings() {
+    insertSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL, String.format("%s%s", getWireMockBasePath(), BASE_URL));
+    flushCache();
+  }
+  
+  private void deletePtvSettings() {
+    deleteSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL);
+  }
+  
+  private void createGtfsSettings(String organizationId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "createGtfsSettings" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)